content/kubermatic/master/tutorials_howtos/applications/create-application-catalogue/_index.en.md

```diff
@@ -0,0 +1,248 @@
+This guide targets KKP Admins and details adding Applications to a catalogue, so they can be installed by Cluster Admins.
+Create permissions on the KKP master cluster are required to complete it.
+
+Before an Application is available for install, its installation- and metadata need to be added to the KKP Master Cluster. From the master, they will be automatically replicated to all KKP Seed Clusters.
+
+{{< figure src="application-catalogue.png" title="Example of a populated catalogue" >}}
+
+To organize its catalogue, KKP makes use of a Custom Kubernetes Resource Type called `ApplicationDefinition`. This ensures Kubernetes-native management and full GitOps compatibility.
+Additionally this mechanism can be used to ensure that only approved Applications can be deployed into a cluster.
+
+## Creating an ApplicationDefinition
+
+An ApplicationDefinition represents a single Application and contains all of its versions.
+Each ApplicationDefinition maps to one Application in the UI.
+It consists of three parts: metadata on the Application itself, templating method, and the templating source.
+These will be described in more detail in the subsequent paragraphs; For a complete reference, refer to the  [ApplicationDefinition Reference](#applicationdefinition-reference) section.
+Currently ApplicationDefinitions need to be created by hand, but we plan to include a UI importer in an upcoming release.
+
+```yaml
+# Example of an ApplicationDefinition
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationDefinition
+metadata:
+  name: prometheus
+spec:
+  description: Prometheus is a monitoring system and time series database.
+  method: helm
+  versions:
+  - template:
+      source:
+        helm:
+          chartName: prometheus
+          chartVersion: 15.10.4
+          url: https://prometheus-community.github.io/helm-charts
+    version: 2.36.2
+  - template:
+      source:
+        git:
+          path: charts/prometheus
+          ref:
+            branch: master
+            remote: https://github.com/prometheus-community/helm-charts
+    version: dev
+```
+
+### Application Metadata
+
+Application Metadata is mainly used for presentation and does not affect how an application is deployed:
+
+- `metadata.name` -> name of the application which will be displayed in the UI
+- `spec.versions[].version` -> Version which is displayed in the UI; Each entry of this field must be unique. We strongly recommend to set this to the version of the Application itself not of the chart.
+
+### Templating Method
+
+Currently the `helm` templating method is supported exclusively.
+
+### Templating Source
+
+Source of the Manifests that should be templated. This controls from where your Application manifests should be fetched. You can combine multiple sources (e.g. helm & git) within one ApplicationDefinition.
+A common use-case for combining is to make stable versions (via helm) and a development version (via git) available at the same time.
+
+#### Helm Source
+
```

Comment:
i would introduce here the 2 options HTTP and OCI by adding a description:

```
This source download the chart from a Helm [HTTP repository](https://helm.sh/docs/topics/chart_repository/) or and [OCI repository](https://helm.sh/blog/storing-charts-in-oci/#helm)
```

And changing `Currently the best way to obtain chartName and chartVersion  for an HTTP repository is to make use of helm search:`